src/dtos/comment.dto.ts

```diff
@@ -0,0 +1,56 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { isNotEmpty, IsNotEmpty, IsString, IsUrl, isURL } from 'class-validator';
```

Comment:
```suggestion
import { IsNotEmpty, IsString, IsUrl } from 'class-validator';
```
isNotEmpty e isURL estão sendo importado duas vezes.